src/network/service.rs

```diff
@@ -185,6 +188,13 @@ where
                 // TODO: `false` here means we don't insert ourselves in the DHT, which is the polite thing to do for as long as Kad isn't implemented
                 inbound_allowed: false,
             }))
+            .chain(iter::once(libp2p::ConfigRequestResponse {
+                name: format!("/{}/sync/warp", chain.protocol_id),
+                max_request_size: 1024 * 1024,
```

Comment:
The request is just a hash, right?

```suggestion
                max_request_size: 32,
```